lib/topological_inventory/openshift/operations/core/service_catalog_client.rb

```diff
@@ -43,6 +46,7 @@ def wait_for_provision_complete(name, namespace)
               logger.info("#{event.involvedObject.name}: message [#{event.message}] reason [#{event.reason}]")
               next unless %w[ProvisionedSuccessfully ProvisionCallFailed].include?(event.reason)
 
+              update_task(task_id, :state => "running", :status => "ok", :context => event.message)
```

Comment:
@eclarizio The event.reason will be different when we are just getting status messages, line #47 is skipping all of these messages

![Screen-Shot-2019-04-24-at-2 51 55-PM](https://user-images.githubusercontent.com/6452699/56834679-1588e700-6840-11e9-81a4-744a2c123ab7.png)